controllers/routemonitor/routemonitor.go

```diff
@@ -99,14 +100,24 @@ func (r *RouteMonitorReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 		return utilreconcile.Stop()
 	}
 
-	log.V(2).Info("Entering CreateServiceMonitorResource")
+	log.V(2).Info("Entering EnsureServiceMonitorResourceExists")
 	res, err = r.EnsureServiceMonitorResourceExists(ctx, routeMonitor)
 	if err != nil {
 		return utilreconcile.RequeueWith(err)
 	}
 	if res.ShouldStop() {
 		return utilreconcile.Stop()
 	}
+
+	log.V(2).Info("Entering EnsurePrometheusRuleResourceExists")
+	res, err = r.EnsurePrometheusRuleResourceExists(ctx, routeMonitor)
+	if err != nil {
+		return utilreconcile.RequeueWith(err)
+	}
+	if res.ShouldStop() {
+		return utilreconcile.Stop()
+	}
```

Comment:
this can be removed I assume?

Reply:
it can be removed.
when we moved this section up before we almost forgot to add the `res.ShouldStop`, so it might cause issues later

Reply:
will do then